readme.md

```diff
@@ -12,69 +12,59 @@ This is the official ZURB Template for use with [Foundation for Sites](http://fo
 - Built-in BrowserSync server
 - For production builds:
   - CSS compression
-  - JavaScript compression
+  - JavaScript module bundling with webpack
   - Image compression
 
 ## Installation
 
 To use this template, your computer needs:
 
-- [NodeJS](https://nodejs.org/en/) (Up to Version 8 only, Tested with 8.12.0)
+- [NodeJS](https://nodejs.org/en/) (Version 6 or greater recommended, tested with 6.11.4 and 8.12.0)
 - [Git](https://git-scm.com/)
 
-### Setup
+This template can be installed with the Foundation CLI, or downloaded and set up manually.
 
-To set up the template, first download it.
+### Using the CLI
 
-Then open the folder in your command line, and install the needed dependencies:
+Install the Foundation CLI with this command:
 
 ```bash
-cd projectName
-yarn
+npm install foundation-cli --global
 ```
 
-Now `cd` to your project name and run `npm start` or `foundation watch` to run Gulp. Your finished site will be created in a folder called `dist`, viewable at this URL:
```

Comment:
should say 
Now `cd` to your project name and run `yarn start`  to run Gulp. While developing, you can view your project at http://localhost:8000. Your finished site will be created in a folder called `dist`.

Reply:
This is basically what is being said in `### Manual setup` below:

> Finally, run `yarn start` to run Gulp. Your finished site will be created in a folder called `dist`, viewable at this URL:
> ```
> http://localhost:8000
> ```